ui/app/templates/components/job-version.hbs

```diff
@@ -3,55 +3,110 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<div class="boxed-section-head is-light inline-definitions">
-  Version #{{this.version.number}}
-  <span class="bumper-left pair is-faded">
-    <span class="term">Stable</span>
-    <span class="badge is-light is-faded" data-test-version-stability><code>{{this.version.stable}}</code></span>
-  </span>
-  <span class="pair is-faded">
-    <span class="term">Submitted</span>
-    <span data-test-version-submit-time class="submit-date">{{format-ts this.version.submitTime}}</span>
-  </span>
-  <div class="pull-right">
-    {{#unless this.isCurrent}}
-      {{#if (can "run job" namespace=this.version.job.namespace)}}
-        <TwoStepButton
-          data-test-revert-to
-          @classes={{hash
-            idleButton="is-warning is-outlined"
-            confirmButton="is-warning"}}
-          @alignRight={{true}}
-          @idleText="Revert Version"
-          @cancelText="Cancel"
-          @confirmText="Yes, Revert Version"
-          @confirmationMessage="Are you sure you want to revert to this version?"
-          @inlineText={{true}}
-          @fadingBackground={{true}}
-          @awaitingConfirmation={{this.revertTo.isRunning}}
-          @onConfirm={{perform this.revertTo}} />
+<section class="job-version">
+  <div class="boxed-section {{if this.version.taggedVersion "tagged"}}">
+    <header class="boxed-section-head is-light inline-definitions">
+      Version #{{this.version.number}}
+      <span class="bumper-left pair is-faded">
+        <span class="term">Stable</span>
+        <span class="badge is-light is-faded" data-test-version-stability><code>{{this.version.stable}}</code></span>
+      </span>
+      <span class="pair is-faded">
+        <span class="term">Submitted</span>
+        <span data-test-version-submit-time class="submit-date">{{format-ts this.version.submitTime}}</span>
+      </span>
+      <div class="pull-right">
+        {{#if this.version.diff}}
+          <Hds::Button
+            class="is-light is-small"
+            @size="small"
+            @text="{{if this.isOpen "Hide" "See"}} {{this.changeCount}} {{pluralize "Change" this.changeCount}}"
+            @color="tertiary"
+            @icon={{if this.isOpen "chevron-up" "chevron-down"}}
+            @iconPosition="trailing"
+            {{on "click" (action "toggleDiff")}}
+          />
+        {{else}}
+          <div class="is-fixed-width is-size-7 has-text-centered">No Changes</div>
+        {{/if}}
+      </div>
+    </header>
+    {{#if this.isOpen}}
+      <div class="boxed-section-body is-dark">
+        <JobDiff @diff={{this.version.diff}} @verbose={{this.verbose}} />
+      </div>
+    {{/if}}
+    <footer class="boxed-section-foot {{if this.isEditing "editing"}}">
+      {{#if this.isEditing}}
+        <form class="tag-options" autocomplete="off" {{on "submit" (action "saveTag")}}>
+          {{! template-lint-disable no-down-event-binding }}
```

Comment:
This is to prevent `{{on "keydown" (action this.handleKeydown)}}` from throwing a linting error. This method watches for an `ESC` keypress to close edit mode.

I have a difference of opinion with our linter, expressed better than I can do it at https://x.com/ID_AA_Carmack/status/1787850053912064005